Helper/Data.php

```diff
@@ -103,6 +103,21 @@ class Data extends AbstractHelper
     const XML_PATH_PRODUCT_UPDATES = 'nosto/flags/product_updates';
 
     /**
+     * Path to the configuration object for customized tag1
+     */
+    const XML_PATH_TAG1 = 'nosto/attributes/tag1';
```

Comment:
Do we really need separate constants for each path? I'd go with M1 approach: https://github.com/Nosto/nosto-magento/blob/develop/app/code/community/Nosto/Tagging/Helper/Data.php#L50